README.md

```diff
@@ -34,11 +24,31 @@ based on the [Bulletproofs](https://doc.dalek.rs/bulletproofs/index.html) zero-k
 * [Spacesuit README](spacesuit/README.md)
 * [Cloak specification](spacesuit/spec.md)
 
+### [Musig](musig)
+
+A pure Rust implementation of the [MuSig signature aggregation scheme](https://eprint.iacr.org/2018/068) by Maxwell, Poelstra, Seurin and Wuille.
+
+* [Musig README](musig/README.md)
+
 ### [Keytree](keytree)
 
 A _key blinding scheme_ for deriving hierarchies of public keys for [Ristretto](https://ristretto.group)-based signatures.
 
 * [Specification](keytree/keytree.md)
 
+### [Token](token)
+
+An API for issuing assets using ZkVM.
+
+
```

Comment:
nit but I think you have an extra newline here

Reply:
👍